v3/fleet-local/control/flight-director@.service

```diff
@@ -39,6 +39,11 @@ ExecStart=/usr/bin/sh -c "/usr/bin/docker run \
   -e FD_LOG_MARATHON_API_CALLS=`etcdctl get /FD/FD_LOG_MARATHON_API_CALLS` \
   -e FD_MARATHON_MASTER=`etcdctl get /FD/FD_MARATHON_MASTER` \
   -e FD_MESOS_MASTER=`etcdctl get /FD/FD_MESOS_MASTER` \
+  -e GITHUB_ALLOWED_TEAMS=`etcdctl get /FD/GITHUB_ALLOWED_TEAMS` \
```

Comment:
@iderdik @chr0n1x I see values on the behancecompute cluster, not sure having them removed broke anything or not anymore. We tried so many things the other day ...

```
core@ip-10-74-185-32 stage-behancecompute-us-east-1c-worker ~ $ etcdctl ls /FD
/FD/FD_CHRONOS_MASTER
/FD/FD_DB_PASSWORD
/FD/FD_DB_USERNAME
/FD/FD_DOCKERCFG_LOCATION
/FD/FD_DEBUG
/FD/FD_EVENT_PORT
/FD/FD_KV_SERVER
/FD/FD_AIRLOCK_PUBLIC_KEY
/FD/GITHUB_ALLOWED_TEAMS
/FD/GITHUB_CLIENT_ID
/FD/GITHUB_CLIENT_SECRET
/FD/FD_API_SERVER_PORT
/FD/FD_DB_DATABASE
/FD/FD_EVENT_INTERFACE
/FD/FD_LOG_LOCATION
/FD/FD_LOG_MARATHON_API_CALLS
/FD/FD_MESOS_MASTER
/FD/GITHUB_TOKEN_URL
/FD/FD_DB_ENGINE
/FD/FD_DB_PATH
/FD/FD_FIXTURES
/FD/FD_LOG_LEVEL
/FD/GITHUB_API
/FD/FD_MARATHON_MASTER
/FD/AUTHORIZER_TYPE
core@ip-10-74-185-32 stage-behancecompute-us-east-1c-worker ~ $
```